source/agora/consensus/data/UTXOSet.d

```diff
@@ -27,6 +27,56 @@ import std.file;
 
 mixin AddLogger!();
 
+/// The structure of spendable transaction output
+public struct UTXOSetValue
+{
+    /// Block height
+    ulong height;
+
+    /// Transaction type
+    TxType type;
+
+    /// Previous transaction type
+    TxType prev_type;
```

Comment:
What is this field for ?

Reply:
Type of previous transaction.
To verify that the freeze is a UTXO just melted.

Reply:
Please refer to the table below.
<img width="896" alt="Freezing status" src="https://user-images.githubusercontent.com/11639939/64929810-bd419880-d865-11e9-9165-e6c068a7527e.png">